WarehouseManagement.Common/Statuses/ZoneEntryStatuses.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace WarehouseManagement.Common.Statuses
+{
+    public static class ZoneEntryStatuses
+    {
+        public const string Waiting = "Waiting";
+        public const string Processing = "Processing";
+        public const string Finished = "Finished";
+    }
```

Comment:
wouldnt an enum make more sense?

Reply:
Yeah, should also work, I will handle it.